receiver/flinkmetricsreceiver/metadata.yaml

```
@@ -0,0 +1,328 @@
+name: flinkmetricsreceiver
+
+resource_attributes:
+  # These resource attributes are Flinks system scope variables, which contains context information about metrics. These are required to uniquely identify incoming metrics as the same job can run multiple times concurrently. See https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/ops/metrics/#system-scope for more information.
+  host.name:
+    description: The host name.
+    type: string
+  flink.taskmanager.id:
+    description: The taskmanager ID.
+    type: string
+  flink.job.name:
+    description: The job name.
+    type: string
+  flink.task.name:
+    description: The task name.
+    type: string
+  flink.subtask.index:
+    description: The subtask index.
+    type: string
+  flink.resource.type:
+    description: The flink scope type in which a metric belongs to.
+    type: string
+    enum: [ jobmanager, taskmanager ]
+
+attributes:
+  operator_name:
```

This can be a followup issue, but I think this should probably be user-visible as `name`.

```suggestion
  operator_name:
    value: name
```